src/main/java/org/threadly/litesockets/utils/MergedByteBuffers.java

```diff
@@ -17,13 +17,13 @@
  */
 public class MergedByteBuffers {
```

Comment:
One thing I realized...this class is not thread safe, where a ByteBuffer typically is considered thread safe.  I think at a minimum we should document that.